nilearn/_utils/cache_mixin.py

```diff
@@ -195,6 +195,21 @@ class CacheMixin:
 
     """
 
+    def _fit_cache(self):
+        """Set attributes during estimator fit.
+
+        _shelving : bool
+            Used during calls to the method _cache
+            to return a joblib MemorizedResult,
+            callable by a .get() method,
+            instead of the return value of func.
```

Comment:
`_shelving` is not a parameter of `_fit_cache`. Is there a reason to document it here?